activities/tool-management/mailing lists.md

```diff
@@ -0,0 +1,61 @@
+---
+type: resource, tooling
+---
+
+# Mailman mailing lists
+
+Mailman is free software for managing electronic mail discussion and e-newsletter lists. Mailman is integrated with the web, making it easy for users to manage their accounts and for list owners to administer their lists. Mailman supports built-in archiving, automatic bounce processing, content filtering, digest delivery, spam filters, and more.
```

Comment:
This reads like an advertisement for Mailman. Deleting the second and third sentences and replacing them with a statement of the value it provides us, or why we use it, might make it seem less like an ad.